packages/swapper/src/cowswap-utils/index.ts

```diff
@@ -181,15 +182,15 @@ export const getAffiliateAppDataFragmentByChainId = ({
   affiliateBps,
   chainId,
 }: {
-  affiliateBps: string
+  affiliateBps: BigNumber.Value
```

Comment:
Speaking `BigNumber.Value` across the board instead of string since the constant bps is a number - also note no undefined across the board since affiliate bps are always defined now (even if zero'd out, which they should never be really, but technically possible)